app/src/main/scala/org/alephium/explorer/persistence/schema/CustomJdbcTypes.scala

```diff
@@ -101,4 +102,10 @@ object CustomJdbcTypes {
       IntervalType.unsafe
     )
 
+  @SuppressWarnings(Array("org.wartremover.warts.OptionPartial"))
+  implicit lazy val appStateValue: JdbcType[AppState.Value] =
+    MappedJdbcType.base[AppState.Value, ByteString](
+      appState => AppState.Value.serialize(appState.value),
+      input    => AppState.Value.deserialize(input).toOption.get
+    )
```

Comment:
Similar to above, instead of `.toOption.get` should we use `fold` or `match` instead to get the actual error in the call stack in case of failure?

Reply:
removed in last commit